content/doc/developer/views/table-to-div-migration.adoc

```diff
@@ -14,6 +14,32 @@ as part of jira:JENKINS-56109[].
 For 99% of plugins they do not require any changes at all as they just use the standard jelly tags
 in Jenkins for their forms, such as `f:entry`, `f:textbox`, `f:checkbox`.
 
+== Some plugins are known to be broken
+
+If you're experiencing problems,
+please check the link:https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=207410[list of known broken plugins dashboard].
+If you're using any of these plugins, please disable them (or wait to upgrade).
+
+If you're experiencing problems (form controls not working, the save button not working)
+you can minimize your jenkins installation to identify which plugin is the problem.
+
+. Set up minimal jenkins
+
+    curl https://issues.jenkins.io/secure/attachment/53350/53350_jenkins-basith-16112020.txt | sort > abdul-bas-plugins.txt
+    cat abdul-bas-plugins.txt | cut -d ':' -f 1 | sort > plugins.filtered.txt # remove versions as CLI complains when old versions are there
```

Comment:
Why this specific set of plugins?  That list certainly contains many different plugins than the set of plugins that I use and many different plugins than are used on ci.jenkins.io.  Isn't it much more difficult to give them a list of plugins that they are not using and have them remove plugins than to tell them to generate the list of plugins they are using and disable plugins while searching for the issue?
```suggestion
    curl https://issues.jenkins.io/secure/attachment/53350/53350_jenkins-basith-16112020.txt | sort > abdul-bas-plugins.txt
    cat abdul-bas-plugins.txt | cut -d ':' -f 1 | sort > plugins.filtered.txt # remove versions as CLI complains when old versions are there
```

Reply:
@timja wrote it in https://issues.jenkins.io/browse/JENKINS-64072  -- I believe I've copied it correctly.

Reply:
this was an example for triaging someone else's issue,

for someone doing their own issue, they can either reproduce on an existing one they have,

or use the API to produce a plugin list that can be used to setup a new instance,

```
JENKINS_HOST=username:password@myhost.com:port
curl -sSL "http://$JENKINS_HOST/pluginManager/api/xml?depth=1&xpath=/*/*/shortName|/*/*/version&wrapper=plugins" | perl -pe 's/.*?<shortName>([\w-]+).*?<version>([^<]+)()(<\/\w+>)+/\1 \2\n/g'|sed 's/ /:/' | cut -d ':' -f 1 
```